src/textures/DataTexture2DArray.js

```diff
@@ -3,16 +3,13 @@ import { ClampToEdgeWrapping, NearestFilter } from '../constants.js';
 
 class DataTexture2DArray extends Texture {
 
-	constructor( data = null, width = 1, height = 1, depth = 1 ) {
+	constructor( data = null, width = 1, height = 1, depth = 1, format, type, mapping, wrapS, wrapT, wrapR = ClampToEdgeWrapping, magFilter = NearestFilter, minFilter = NearestFilter, anisotropy, encoding ) {
```

Comment:
I don't think we should continue this pattern. Better to keep these constructors simple.

Reply:
The disadvantages of not having ctor parameters is the fact that we can't use the mechanism of default parameters when setting up a texture. Instead of one-liner we need this in the render target classes:
```js
if ( options.format !== undefined ) this.texture.format = options.format;
if ( options.type !== undefined ) this.texture.type = options.type;
if ( options.mapping !== undefined ) this.texture.mapping = options.mapping;
if ( options.wrapS !== undefined ) this.texture.wrapS = options.wrapS;
if ( options.wrapT !== undefined ) this.texture.wrapT = options.wrapT;
if ( options.wrapR !== undefined ) this.texture.wrapR = options.wrapR;
if ( options.magFilter !== undefined ) this.texture.magFilter = options.magFilter;
if ( options.minFilter !== undefined ) this.texture.minFilter = options.minFilter;
if ( options.anisotropy !== undefined ) this.texture.anisotropy = options.anisotropy;
if ( options.encoding !== undefined ) this.texture.encoding = options.encoding;
```
But I understand that the long signatures of ctor are not pretty, too.